moses/moses/main/populate_atomspace.cc

```diff
@@ -0,0 +1,62 @@
+#include <moses/data/table/table.h>
+#include <moses/data/table/table_io.h>
+#include <opencog/atomspace/AtomSpace.h>
+#include <moses/utils/value_key.h>
+#include <opencog/atoms/base/Link.h>
+#include "populate_atomspace.h"
+#include <opencog/atoms/value/Value.h>
+
+namespace opencog { namespace moses {
+
+void populate(AtomSpace *as, ITable &itable) {
+
+	std::vector<multi_type_seq>::const_iterator it, end;
+	int col_size = itable.get_types().size();
+	for (int i = 0; i < col_size; i++) {
+		id::type_node col_type = itable.get_types().at(i);
+		int row_size = itable.size();
```

Comment:
Same comment for `row_size`, `rows_size` would be better.

Reply:
went by n_rows like n_columns